_inc/client/search/index.jsx

```diff
@@ -105,6 +105,7 @@ export const SearchResults = ( {
 		let isPro = 'scan' === element[0]
 				 || 'akismet' === element[0]
 				 || 'backups' === element[0]
+				 || 'videopress' === element[0]
```

Comment:
Cruft from master having been merged in before this was rebased onto branch-4.4

Honestly, @samhotchkiss -- does it have to be rebased onto branch-4.4?  Couldn't it just be rebased onto Master, but merged into 4.4 and master both when approved?  Or smush-merged into master, then cherry-picked across?